### ISSUE TITLE
instagram verification fails

### ISSUE THREAD

**Issue by diplix** [aaronpk/OwnYourGram#28]
ownyourgram or indieauth adds a trailing slash to my domain (http://wirres.net) even if i try to auth with ownyourgram as http://wirres.net — instagram however removes the trailing slash that i enter into the `external_url` field.

also adding http://wirres.net/ to my bio fails, possibly due to caching the instagram responses.

so i can’t authenticate with ownyourgram right now, or only after i wait till the instgram response cache empties.

**Comment by aaronpk** [comment]
This should be fixed with a more recent version of the IndieAuth client that's being used right now. Feel free to open a new issue if it's still a problem. Thanks!
